openpower/package/sbe-odyssey/sbe-odyssey.mk

```diff
@@ -29,8 +29,10 @@ endef
 
 define SBE_ODYSSEY_INSTALL_IMAGES_CMDS
 	mkdir -p $(STAGING_DIR)/ody_binaries
+	mkdir -p $(STAGING_DIR)/ody_debug_tools
```

Comment:
I would suggest using "poz" instead of "ody", since all tools we are planning to make it common and some simics scripts already made it common.

What about the dd level?

Reply:
yeah will make poz. dd level will see later